### PR TITLE
{vis}[GCC/8.2.0-2.31.1] FLTK v1.3.5

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/23d065fe00a86a0df0b3c05a0f407f3e for a full test report.

File: easybuild/easyconfigs/f/FLTK/FLTK-1.3.5-GCC-8.2.0-2.31.1.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+name = 'FLTK'
+version = '1.3.5'
+
+homepage = 'http://www.fltk.org'
+description = """FLTK is a cross-platform C++ GUI toolkit for UNIX/Linux (X11), Microsoft Windows,
+ and MacOS X. FLTK provides modern GUI functionality without the bloat and supports 3D graphics via OpenGL
+ and its built-in GLUT emulation."""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
```
Comment (Micket) [inline easybuild/easyconfigs/f/FLTK/FLTK-1.3.5-GCC-8.2.0-2.31.1.eb]: I suspect you have a good reason, but why not GCCcore?

Comment (akesandgren) [reply easybuild/easyconfigs/f/FLTK/FLTK-1.3.5-GCC-8.2.0-2.31.1.eb]: For the simple reason that this is getting into the GUI performance realm. And for that i prefer the upper level compiler when using intel based toolchains.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/867d9083b5c00f2e9f0e9e78f15313a3 for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @akesandgren!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/26224ad1d6b04a8a1be37083edfc1ce2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2039.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f798ca728e6725f9b09826084fd81f3c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2406.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d43a66c8d4c132dcdf2f79cf63ae28d9 for a full test report.

**Pull request merged**